### PR TITLE
[13.0][ADD] base_module_uninstall_batch

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Adds option to remove modules in batch.

### CODE REVIEW

File: base_module_remove_batch/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Module Remove Batch",
+    "summary": """
+        Add an option to remove modules in batch.
+    """,
+    "version": "12.0.2.1.0",
```
Comment (dalonsod) [inline base_module_remove_batch/__manifest__.py]: Wrong version

File: base_module_remove_batch/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "license": "LGPL-3",
+    "category": "Base",
+    "author": "Solvos",
+    "website": "https://github.com/solvosci/slv-helpdesk",
```
Comment (dalonsod) [inline base_module_remove_batch/__manifest__.py]: Wrong website

File: base_module_remove_batch/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
```
Comment (dalonsod) [inline base_module_remove_batch/__manifest__.py]: Wrong year

File: base_module_remove_batch/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    "version": "12.0.2.1.0",
+    "license": "LGPL-3",
+    "category": "Base",
+    "author": "Solvos",
+    "website": "https://github.com/solvosci/slv-helpdesk",
+    "depends": [
+        "base",
+    ],
+    "data": [
+        "data/ir_action_data.xml",
```
Comment (dalonsod) [inline base_module_remove_batch/__manifest__.py]: I'm not sure if data such as a server action should be placed in a `data` folder. `data` folder is often used for master initialization data. Maybe a server action should be treated like other structures (views, menus, client actions,...) that are placed under `views` folder instead.

**Review by dalonsod — CHANGES_REQUESTED** [review]
Only code review, see comments. Also,
- is it possible to fire an error if there's no uninstallable addons in the selection?
- re-thinking about addon naming (I'm sorry, but this name was suggested by me), now I've changed opinion: a better name should be `base_module_uninstall_batch`, because we're trying to uninstall modules, not removing them from Odoo bdd. Other possibility is adding another action "Uninstall and remove modules"; then, the current module name could be preserved. What do you think about it?

File: base_module_remove_batch/__manifest__.py (hunk 1 above)
Comment (dalonsod) [reply base_module_remove_batch/__manifest__.py]: Still wrong version... this addon is for v13, isn't it?

Comment (ChristianSantamaria) [reply base_module_remove_batch/__manifest__.py]: Yes, sorry, I had done it in v12 but in v13 it still works

**Comment by ChristianSantamaria** [comment]
Yes, you are right, it is a more appropriate name if in the future we make it eliminate the modules we will change the name again.

**Comment by ChristianSantamaria** [comment]
> is it possible to fire an error if there's no uninstallable addons in the selection?

In the event that one or more selected modules are already uninstalled, odoo will show this warning message
imagen

File: base_module_uninstall_batch/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Module Uninstall Batch",
+    "summary": """
+        Adds option to uninstall modules in batch.
+    """,
+    "version": "13.0.0.1.0",
```
Comment (dalonsod) [inline base_module_uninstall_batch/__manifest__.py]: Wrong version
```suggestion
    "version": "13.0.1.0.0",
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Locally tested, LGTM.
Please add `i18n` files, generate the `README.rst` and `index.html` files and squash commits

**Review by dalonsod — APPROVED** [review]
LGTM

**Comment by dalonsod** [comment]
In a future version we could add a second button called "Uninstall & remove", but it's not mandatory for now. Merging...

**Pull request merged**